### PR TITLE
Update English and German documentation about item variations and editing information in Plenty

### PR DESCRIPTION
**Pull request by jathmel**
- Correct the instructions about linking Item Variations with Plentymarkets (using main variation, item inheritence).
- Update the documentation to no longer mention the "edit button" in the places where Plentymarkets has removed it

### CODE REVIEW

**Review by jhoule86 — CHANGES_REQUESTED** [review]
Some comments based on previous conversations.

Also, I think you missed this section of the docs, where the main information on Variation linking lives: https://github.com/wayfair-contribs/plentymarkets-plugin/blob/main/meta/documents/user_guide/en/initial_setup.md#5-making-items-available-for-sale-on-wayfair

File: meta/documents/user_guide/de/initial_setup.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Die Schritte Autorisierung sind wie folgt:
 
     ![linked clients](../../../images/de/plugin_sets_linked_clients.png)
 
-3. Klicken Sie auf die Schaltfläche `bearbeiten` ![pencil button](../../../images/common/button_pencil.png) für das gewünschte Plugin-Set.
+3. Klicken Sie auf das gewünschte Plugin-Set..
```
Comment (mcloughlin77) [inline meta/documents/user_guide/de/initial_setup.md:36]:
```suggestion
3. Klicken Sie auf das gewünschte Plugin-Set.
```
I assume the double periods wasn't intentional and it doesn't mean something in German?

File: meta/documents/user_guide/de/initial_setup.md [hunk 2]
```diff
@@ -79,33 +79,31 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
-2. Suchen Sie nach Artikeln und öffnen Sie sie
+2.  Suchen Sie nach Artikeln und öffnen Sie sie **`Stellen Sie sicher, dass auf dem Artikel die **`Hauptvariante`** steht`** siehe Bild unten
 
-3. **Klicken Sie für jedes Element** auf `Varianten` und öffnen Sie diese.
+    ![item main variation](../../../images/de/item_main_variation.png)
 
-4. **Für jede Variante**:
-
-    1. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
+3.Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
```
Comment (mcloughlin77) [inline meta/documents/user_guide/de/initial_setup.md:86]:
```suggestion
3. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
```

File: meta/documents/user_guide/de/initial_setup.md [hunk 3]
```diff
@@ -79,33 +79,31 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
 
-        5. Beachten Sie, dass in `Märkte` jetzt eine `Wayfair` -Reihe vorhanden ist
+    5. Beachten Sie, dass in `Märkte` jetzt eine `Wayfair` -Reihe vorhanden ist
 
-            ![adding wayfair market to variation](../../../images/de/variation_wayfair_market_added.png)
+        ![adding wayfair market to variation](../../../images/de/variation_wayfair_market_added.png)
 
 
-    3. Klicken Sie auf die Schaltfläche `Speichern` neben der Variations-ID (nicht auf die obere Schaltfläche für den Artikel).
+5 Klicken Sie auf die Schaltfläche `Speichern` neben der Artikel-ID (nicht auf die obere Schaltfläche für den Artikel).
```
Comment (mcloughlin77) [inline meta/documents/user_guide/de/initial_setup.md:106]:
```suggestion
5. Klicken Sie auf die Schaltfläche `Speichern` neben der Artikel-ID (nicht auf die obere Schaltfläche für den Artikel).
```

File: meta/documents/user_guide/en/initial_setup.md [hunk 4]
```diff
@@ -81,37 +81,37 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
 
 1. From the main Plentymarkets page, go to `Item` >> `Edit item`
 
-2. Search for item(s) and open them
+2. Search for item(s) and open them Ensure that item says **`main variation`** see image below
```
Comment (jhoule86) [inline meta/documents/user_guide/en/initial_setup.md:84]: The fragment at the end of this is probably meant to be a new step or sub-step?
Also, there's no punctuation or separation between ideas.

File: meta/documents/user_guide/en/initial_setup.md [hunk 5]
```diff
@@ -81,37 +81,37 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
...
 
-        5. Observe that a `Wayfair` row now exists in `Markets`
+    5. Observe that a `Wayfair` row now exists in `Markets`
 
-            ![adding wayfair market to variation](../../../images/en/variation_wayfair_market_added.png)
+        ![adding wayfair market to variation](../../../images/en/variation_wayfair_market_added.png)
 
 
-    3. Click the `Save` left of the Variation `ID`, below the `Global` tab (not the higher-up button for the Item):
+5. Click the `Save` left of the `Item ID`, below the `Global` tab (not the higher-up button for the Item):
```
Comment (jhoule86) [inline meta/documents/user_guide/en/initial_setup.md:110]: The user is modifying things at the `Variation` level, and the screenshot currently points to the `Variation` save button. Do we really want to change this to `Item`?

File: meta/documents/user_guide/de/initial_setup.md [hunk 6]
```diff
@@ -79,33 +79,31 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
 
-3. **Klicken Sie für jedes Element** auf `Varianten` und öffnen Sie diese.
+    ![item main variation](../../../images/de/item_main_variation.png)
 
-4. **Für jede Variante**:
-
-    1. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
+3. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
 
         ![variation active](../../../images/de/variation_active_field.png)
```
Comment (mcloughlin77) [inline meta/documents/user_guide/de/initial_setup.md:88]: This doesn't show up in the rich view

File: meta/documents/user_guide/de/initial_setup.md [hunk 7]
```diff
@@ -79,33 +79,31 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
+    5. Beachten Sie, dass in `Märkte` jetzt eine `Wayfair` -Reihe vorhanden ist
 
-            ![adding wayfair market to variation](../../../images/de/variation_wayfair_market_added.png)
+        ![adding wayfair market to variation](../../../images/de/variation_wayfair_market_added.png)
 
 
-    3. Klicken Sie auf die Schaltfläche `Speichern` neben der Variations-ID (nicht auf die obere Schaltfläche für den Artikel).
+5. Klicken Sie auf die Schaltfläche `Speichern` neben der Artikel-ID (nicht auf die obere Schaltfläche für den Artikel).
 
         ![variation saving](../../../images/common/variation_save.png)
```
Comment (mcloughlin77) [inline meta/documents/user_guide/de/initial_setup.md:108]: This doesn't show up in the rich view.

File: meta/documents/user_guide/en/initial_setup.md [hunk 8]
```diff
@@ -81,37 +81,37 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
...
 
-        ![variation saving](../../../images/common/variation_save.png)
+    The ![variation saving](../../../images/common/variation_save.png)
```
Comment (jhoule86) [inline meta/documents/user_guide/en/initial_setup.md:112]: This `The` is just hanging out next to the image.
Also, note the alt-text says `variation` as well as the filename, then look at your above change.
```suggestion
    ![variation saving](../../../images/common/variation_save.png)

```

File: meta/documents/user_guide/en/settings_guide.md [hunk 9]
```diff
@@ -124,9 +124,11 @@ The optional `Import orders since` setting determines a date on which new Wayfai
 
 ### Send all inventory items to Wayfair?
 The optional `Send all inventory items to Wayfair?` setting determines if Plentymarkets Item Variations must be explicitly declared as eligible for sale on Wayfair or not.
-When `Send all inventory items to Wayfair?` is disabled, selling an Item on Wayfair requires having "Wayfair" present in the `Markets` list of the `Availability` tab of each  Variation to be sold on Wayfair, as seen in this example:
+When `Send all inventory items to Wayfair?` is disabled, selling an Item on Wayfair requires having "Wayfair" present in the `Markets` list of the `Availability` tab of an Item, as seen in this example:
 ![item with Wayfair Market](../../../images/en/item_wayfair_market.png)
 
+**Note:** All variations of an item will inherit from the  main variation, unless **`Inheritance is deactivated`**
```
Comment (jhoule86) [inline meta/documents/user_guide/en/settings_guide.md:130]: Add a hint/image here about how Item/Variation Inheritance is enabled/disabled.

**Review by jhoule86 — CHANGES_REQUESTED** [review]
A few more issues/suggestions.

File: meta/documents/user_guide/de/initial_setup.md (hunk 6 above)
Comment (jhoule86) [reply meta/documents/user_guide/de/initial_setup.md:88]: Yeah this image seems to have gone missing somewhere!

Comment (jhoule86) [reply meta/documents/user_guide/de/initial_setup.md:88]: @jathmel I think you need to remove some of the indentations before this line, in order for the image to get rendered.

**Review by jhoule86 — CHANGES_REQUESTED** [review]

File: meta/documents/user_guide/de/initial_setup.md (hunk 7 above)
Comment (jhoule86) [reply meta/documents/user_guide/de/initial_setup.md:108]: this is another indentation issue, @jathmel.

File: meta/documents/user_guide/en/initial_setup.md (hunk 4 above)
Comment (jathmel) [reply meta/documents/user_guide/en/initial_setup.md:84]: it is more of a note to suppliers, I'll a the note keyword

**Review by mcloughlin77 — DISMISSED** [review]

File: meta/documents/user_guide/de/initial_setup.md [hunk 10]
```diff
@@ -79,36 +79,37 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
 
 1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >> `Artikel bearbeiten`.
 
-2. Suchen Sie nach Artikeln und öffnen Sie sie
+2.  Suchen Sie nach Artikeln und öffnen Sie sie **`Stellen Sie sicher, dass auf dem Artikel die **`Hauptvariante`** steht`** siehe Bild unten
```
Comment (jhoule86) [inline meta/documents/user_guide/de/initial_setup.md:82]: I think there's an extra space character at the beginning of this item now?

Comment (jhoule86) [inline meta/documents/user_guide/de/initial_setup.md:82]: This has the same problem as the English version did - the part starting with the bolded text is hanging out at the end of this step without any punctuations.

File: meta/documents/user_guide/en/initial_setup.md [hunk 11]
```diff
@@ -82,36 +82,38 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
...
 2. Search for item(s) and open them
+    **Note:** Verify item variation  says **`main variation`** see image below
 
-3. **For each item**, click `Variations` and open them
+    ![item main variation](../../../images/en/item_main_variation.png)
 
-4. **For each Variation**:
 
-    1. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability` Section is checked:
+3. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability` Section is checked:
```
Comment (jhoule86) [inline meta/documents/user_guide/en/initial_setup.md:90]: Hm, are we missing a "For each Item" somewhere in here?

File: meta/documents/user_guide/de/initial_setup.md [hunk 12]
```diff
@@ -79,36 +79,37 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
+    ![item main variation](../../../images/de/item_main_variation.png)
 
-4. **Für jede Variante**:
+3. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
 
-    1. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
+    ![variation active](../../../images/de/variation_active_field.png)
 
-        ![variation active](../../../images/de/variation_active_field.png)
+4. Wenn [die Einstellung `Vollständigen Bestand an Wayfair senden?`](settings_guide.md#vollständigen-bestand-an-wayfair-senden) **deaktiviert** ist, wechseln Sie zur Registerkarte `Verfügbarkeit` des Artikels und fügen Sie `Wayfair` zur Liste im Bereich `Märkte` hinzu.
```
Comment (jhoule86) [inline meta/documents/user_guide/de/initial_setup.md:90]: Hm, are we missing a "For each Item" somewhere in here?

File: meta/documents/user_guide/de/initial_setup.md [hunk 13]
```diff
@@ -79,36 +79,37 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
-            ![adding wayfair market to variation](../../../images/de/variation_wayfair_market_added.png)
 
+5. Klicken Sie auf die Schaltfläche `Speichern` neben der Artikel-ID (nicht auf die obere Schaltfläche für den Artikel).
 
-    3. Klicken Sie auf die Schaltfläche `Speichern` neben der Variations-ID (nicht auf die obere Schaltfläche für den Artikel).
+    ![variation saving](../../../images/common/variation_save.png)
 
-        ![variation saving](../../../images/common/variation_save.png)
 
+**Hinweis:** Alle Variationen eines Elements erben von der Hauptvariante, es sei denn, **`Vererbung deaktivieren`**
```
Comment (jhoule86) [inline meta/documents/user_guide/de/initial_setup.md:111]: This sentence seems unfished.
```suggestion
**Hinweis:** Alle Variationen eines Elements erben von der Hauptvariante, es sei denn der **`Vererbung deaktivieren`** Einstellung wurde angewendet.

```

File: meta/documents/user_guide/de/plugin_installation.md [hunk 14]
```diff
@@ -49,7 +49,7 @@ Um die Funktionen des Wayfair-Plugins nutzen zu können, muss es in den aktiven
     * Verwenden Sie optional die Schaltflächen `Mandanten und Plugin-Sets verknüpfen`, um die Verknüpfungen zwischen Mandanten und Plugin-Sets zu ändern.
         ![link plugin sets button](../../../images/de/installation/button_link_plugin_sets.png)
 
-4. Klicken Sie auf die Schaltfläche `Plugin-Set bearbeiten` ![pencil button](../../../images/common/button_pencil.png) für das Plugin-Set, in dem das Wayfair-Plugin installiert werden soll.
+4. Klicken Sie auf das Plugin-Set, das für das Wayfair-Plugin         installiert werden soll.
```
Comment (jhoule86) [inline meta/documents/user_guide/de/plugin_installation.md:52]: This sentence doesn't sound right, and it's got some extra spaces in it.
```suggestion
4. Klicken Sie auf das Plugin-Set, an in es für das Wayfair-Plugin installiert werden.

```

**Review by jhoule86 — CHANGES_REQUESTED** [review]
A few last things to work on.

- I've provided suggestions for the German issues.
- It looks like we've lost the concept of "link each item/Variation." We need to imply that the process should be repeated for everything they want to associate, unless they pick the "all items" setting.

File: meta/documents/user_guide/en/initial_setup.md (hunk 11 above)
Comment (jathmel) [reply meta/documents/user_guide/en/initial_setup.md:90]: I don't think the `for each item` is necessary

File: meta/documents/user_guide/en/initial_setup.md [hunk 15]
```diff
@@ -82,36 +82,40 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
...
 
-    3. Click the `Save` left of the Variation `ID`, below the `Global` tab (not the higher-up button for the Item):
 
-        ![variation saving](../../../images/common/variation_save.png)
+5. Click the `Save` left of the `Variation ID`, below the `Global` tab (not the higher-up button for the Item):
 
+    ![variation saving](../../../images/common/variation_save.png)
+
+**Note:** All variations of an item will inherit from the  main variation, unless **`Inheritance is deactivated`**
+    ![inheritance status](../../../images/en/alternate_inheritance.png)
```
Comment (jhoule86) [inline meta/documents/user_guide/en/initial_setup.md:118]: there should be a line between the text and image in order for it to render correctly.

File: meta/documents/user_guide/de/initial_setup.md [hunk 16]
```diff
@@ -79,37 +79,42 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
...
-    3. Klicken Sie auf die Schaltfläche `Speichern` neben der Variations-ID (nicht auf die obere Schaltfläche für den Artikel).
+5. Klicken Sie auf die Schaltfläche `Speichern` neben der Artikel-ID (nicht auf die obere Schaltfläche für den Artikel).
 
-        ![variation saving](../../../images/common/variation_save.png)
+    ![variation saving](../../../images/common/variation_save.png)
 
 
+**Hinweis:** Alle Variationen eines Elements erben von der Hauptvariante, es sei denn der **`Vererbung deaktivieren`** Einstellung wurde angewendet.
+
+    ![inheritance status](../../../images/de/alternate_inheritance.png)
```
Comment (jhoule86) [inline meta/documents/user_guide/de/initial_setup.md:116]: I think you need to fix the indentation here in order for the image to render.

**Review by jhoule86 — CHANGES_REQUESTED** [review]
still some issues with images and indentation?

**Review by jhoule86 — APPROVED** [review]
LGTM.

**Pull request merged**